### PR TITLE
moveOutCrew function and button

### PR DESCRIPTION
**Pull request by wersal454**
## What type of PR is this.
1. [ ] Bug
2. [ ] Change
3. [x] Enhancement

### What have you changed and why?
Information:

Added a button for the commander to force crew out of the vehicle player is looking at.
Including players.    

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [x] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
Spawn vehicles for every side(REB,OCC,INV,CIV(?))
Open Y menu then go to commander options.
Press the new button while looking at vehicle you want crew to be moved out.
Should only work on REB and CIV crew.
Check the notification on the right side of the screen
Spawn uav like vehicles and do the same thing as above
Crew shouldn't moveout of the vehicle that doesn't belong to the player that pressing the button. I.E. other player bought it.
********************************************************
Notes:
At the moment of creating this PR, button sadly does nothing.

### CODE REVIEW

**Comment by wersal454** [comment]
Now button does display stuff but doesn't seem to do anything.

**Comment by Silence-112** [comment]
Make your PR's a draft if they don't work please

**Comment by Silence-112** [comment]
Also if it forces out players that opens up a huge exploit, so probably not the best to do that

**Comment by wersal454** [comment]
But it is for commander only. Shouldn't commander be able to do such thing?

**Comment by Silence-112** [comment]
Perhaps, still seems like a path for someone to be annoying with it

**Comment by wersal454** [comment]
Noticed that commands like moveout, don't work on uav/ugv type of vehicles. I think it's possible to simply delete the "crew" inside such vehicles. At least through zeus it's possible.

**Comment by wersal454** [comment]
Button is actually working(didn't notice it somehow), but uav/ugv is still a problem

**Comment by wersal454** [comment]
IT WORKS
DA BUTTON WORKS,
only thing i'm unable to test by myself, is what happens if you try to moveout crew from vehicle that belongs to some one else(I mean if player bought it, on enemy vehicles it gives a hint as it should). + I added ability to remove UAV driver and gunner.

**Comment by wersal454** [comment]
@SilenceIsFatto can you test it ?

**Comment by Silence-112** [comment]
Add steps to the PR and i'll test it

File: A3A/addons/core/functions/Base/fn_moveOutCrew.sqf [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+Arguments:
+    <OBJECT> Player who is trying to move out the crew.
+    <OBJECT> cursorObject of the player.
+
+Return Value:
+<NIL> nil
+
+Scope: Server/HC, All calls need to be executed on one machine, using an HC is also possible.
+Environment: Unscheduled, is used to sell vehicles, execution cannot be stacked and exploited.
+Public: No
```
Comment (Silence-112) [inline A3A/addons/core/functions/Base/fn_moveOutCrew.sqf]: Does this need to be changed?

Comment (wersal454) [reply A3A/addons/core/functions/Base/fn_moveOutCrew.sqf]: yep

**Comment by Silence-112** [comment]
I can confirm this works, but still need to test the MP part on a server

**Pull request merged**